packages/components/psammead-podcast-promo/src/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,355 @@
+    font-size: 1rem;
+    line-height: 1.25rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .emotion-14 {
+    font-size: 1rem;
+    line-height: 1.25rem;
+  }
+}
+
+.emotion-12 {
+  color: unset;
+}
+
+.emotion-12:before {
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  content: '';
+  overflow: hidden;
+  z-index: 1;
+}
+
+.emotion-12:visited .podcast-promo__visited {
+  color: #AEAEB5;
+}
+
+.emotion-12:focus .podcast-promo__focus {
+  -webkit-text-decoration: underline;
+  text-decoration: underline;
+}
+
+.emotion-10 {
+  -webkit-clip-path: inset(100%);
+  clip-path: inset(100%);
+  -webkit-clip: rect(1px,1px,1px,1px);
+  clip: rect(1px,1px,1px,1px);
+  height: 1px;
+  overflow: hidden;
+  position: absolute;
+  width: 1px;
+  margin: 0;
+}
+
+.emotion-16 {
+  font-size: 0.9375rem;
+  line-height: 1.125rem;
+  font-family: ReithSans,Helvetica,Arial,sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  max-width: 30rem;
+  color: #6E6E73;
+  margin-top: 0.5rem;
+}
+
+@media (min-width:20rem) and (max-width:37.4375rem) {
+  .emotion-16 {
+    font-size: 0.9375rem;
+    line-height: 1.125rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .emotion-16 {
+    font-size: 0.875rem;
+    line-height: 1.125rem;
+  }
+}
+
+.emotion-18 {
+  width: 12px;
+  color: #6E6E73;
+  fill: currentColor;
+  position: relative;
+  top: 2px;
+  margin-right: 8px;
+}
+
+.emotion-20 {
+  font-size: 0.9375rem;
+  line-height: 1.25rem;
+  font-family: ReithSans,Helvetica,Arial,sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  color: #6E6E73;
+}
+
+@media (min-width:20rem) and (max-width:37.4375rem) {
+  .emotion-20 {
+    font-size: 1rem;
+    line-height: 1.25rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .emotion-20 {
+    font-size: 1rem;
+    line-height: 1.25rem;
+  }
+}
+
+<div>
+  <section
+    class="emotion-26 emotion-27"
+    role="region"
+  >
+    <div
+      class="emotion-4 emotion-5"
+    >
+      <svg
+        class=" emotion-0 emotion-1"
+        version="1.1"
+        viewBox="0 0 32 32"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <g>
+          <path
+            d="M18.7,31h-5.3l-2.3-10.4C12.3,19.5,14,19,16,19s3.7,0.5,4.9,1.7L18.7,31z M22,8.2c-1.7-1.7-3.9-2.5-6.1-2.5s-4.4,0.9-6,2.5 l1.7,1.7c1.2-1.2,2.7-1.8,4.3-1.8s3.1,0.6,4.3,1.8L22,8.2z M25.5,4.9c-2.6-2.7-6.1-4-9.5-4S9.1,2.3,6.5,4.9l1.7,1.7 c2.1-2.1,4.9-3.2,7.7-3.2c2.8,0,5.6,1.1,7.8,3.2L25.5,4.9z M12.4,14c0,2,1.5,3.6,3.6,3.6c2,0,3.6-1.5,3.6-3.6 c0-2.1-1.5-3.6-3.6-3.6C13.9,10.4,12.4,11.9,12.4,14z"
+          />
+        </g>
+      </svg>
+      <h2
+        class="emotion-2 emotion-3"
+      >
+        Подкаст
+      </h2>
+    </div>
+    <div
+      class="emotion-24 emotion-25"
+    >
+      <figure
+        class="emotion-8 emotion-9"
+      >
+        <div
+          class="emotion-6 emotion-7"
+        >
+          <img
+            alt="Что это было?"
+            src="http://ichef.bbci.co.uk/images/ic/3000x3000/p0776f5z.jpg"
```

Comment:
should we use a responsive and lazy loaded image?

Reply:
API lets user pass their attributes to this `img` - `srcset` will be on them if they want to use it

re lazy loading, do we have examples of that being done already, or is this a new feature?

Reply:
ah I think lazy loading is handled in simorgh. We use this which I think is just a component to wrap image elements https://www.npmjs.com/package/react-lazyload

Reply:
aah okay, the way this is set up at the minute won't let people wrap the images in custom elements - will need to update that

Reply:
Updated this image component to just be a basic styled `figure` where you have to pass your own `img` - updated the example and readme with how it can be used with the psammead-placeholder and lazy loader in simorgh